docs/setup-aws-lambda.asciidoc

```diff
@@ -1,85 +1,62 @@
+Both, the APM Lambda Extension and the Java APM Agent are added to your Lambda function as https://docs.aws.amazon.com/lambda/latest/dg/invocation-layers.html[AWS Lambda Layers]. Therefore, you need to add the corresponding Layer ARNs (identifiers) to your Lambda function.
+
+include::{apm-aws-lambda-root}/docs/add-extension/add-extension-layer-widget.asciidoc[]
 
 [float]
-[[aws-lambda-instrumenting]]
-==== Install the Elastic APM Java Agent Layer
+===== Step 3: Configure APM on AWS Lambda
 
-The Java Agent is installed as an AWS Layer. 
+The APM Lambda Extension and the APM Java agent are configured through environment variables on the AWS Lambda function.
 
-1. Pick the right ARN from the https://github.com/elastic/apm-agent-java/releases[ARN table for the APM Java Agent Lambda Layer]. Note: The APM Java Agent Layer needs to be in the _same region_ as your Lambda function.
-2. Navigate to your function in the AWS Console
-3. Scroll to the Layers section and click the _Add a layer_ button
-4. Choose the _Specify an ARN_ radio button
-5. Enter the Version ARN of the APM Java Agent Layer in the _Specify an ARN_ text input
-6. Click the _Add_ button
+For the minimal configuration, you will need the _APM Server URL_ to set the destination for APM data and an **{apm-guide-ref}/secret-token.html[APM Secret Token]**.
+If you prefer to use an APM API key instead of the APM secret token, use the `ELASTIC_APM_API_KEY` environment variable instead of `ELASTIC_APM_SECRET_TOKEN` in the following configuration.
 
-Once the APM Java Agent Layer is in place, you'll need to configure some environment variables _in addition_ to the environment variables you already configured for the APM Lambda extension. 
-Configure the following environment variables for the APM Java Agent Layer:
+include::./lambda/configure-lambda-widget.asciidoc[]
 
-1. (required) `AWS_LAMBDA_EXEC_WRAPPER`: set this variable's value to `/opt/elastic-apm-handler`, so that Lambda starts the runtime with an attached APM Java agent.
-2. (recommended) <<config-application-packages, `ELASTIC_APM_APPLICATION_PACKAGES`>>: setting this may improve cold start times.
-3. (optional) Fine-tune the Java agent with any of the available <<configuration, configuration options>>.
+You can optionally <<configuration, fine-tune the Java agent >> or the {apm-guide-ref}/aws-lambda-config-options.html[configuration of the APM Lambda Extension].
 
 That's it, if you've done all of the above you are set to go!
 Your Lambda function invocations should be traced from now on.
 
+[float]
+[[aws-lambda-features-and-caveats]]
+==== Features & Caveats
+
+The AWS Lambda as a runtime behaves different with respect to some aspect compared to conventional runtimes.
+While most APM and monitoring concepts are applicable to AWS Lambda, there are also a few limitations and specifics.
+Read on to learn more about the Java APM features and caveats on AWS Lambda.
```

Comment:
In general, "read on" is assumed. What about something like this? I'm not sure if I captured the runtime difference correctly.

```suggestion
The AWS Lambda as a runtime behaves differently from conventional runtimes.
While most APM and monitoring concepts apply to AWS Lambda, there are a few differences and limitations to be aware of.
```